supervisor/shared/serial.c

```diff
@@ -175,7 +180,9 @@ bool serial_connected(void) {
     #endif
 
     #if CIRCUITPY_CONSOLE_UART
-    return true;
+    if (_console_uart_rx_timestamp && (_console_uart_rx_timestamp + (60 * 1024 * 5) > port_get_raw_ticks(NULL))) {
```

Comment:
Please breakout the 60 * 1024 * 5 into a macro that is documented with what it means.